tests/src/hashmap.cpp

```diff
@@ -63,3 +63,23 @@ TEST_CASE("hm_set further testing", "[hashmap]")
 	hm_destroy(hashmap, free);
 }
 
+TEST_CASE("hm_remove test", "[hashmap]")
+{
+	void *hashmap = hm_new(HM_SIZE);
```

Comment:
best to not use HM_SIZE but use a for loop to test from size 1 to something like 1024 or something.

you can just put the whole test block in the for loop.

that way you test for more than one size (you could do this for every unittest)